src/components/StaticAppBar/StaticAppBar.js

```diff
@@ -130,6 +132,24 @@ class StaticAppBar extends Component {
   };
 
   componentDidMount() {
+    $.ajax({
+      url:
+        'https://api.susi.ai' +
+        '/aaa/listUserSettings.json?access_token=' +
+        cookies.get('loggedIn'),
+      jsonpCallback: 'pc',
+      dataType: 'jsonp',
+      jsonp: 'callback',
+      crossDomain: true,
+      success: function(data) {
+        let userName = data.settings.userName;
```

Comment:
Can you add a check here if the `settings.`userName` exists or not? because the key won't be coming from backend if the user has no name set.

Reply:
Yes, I added that initially but then I checked the response in that case and the key is coming everytime as an empty string in case no username is set which is handled while rendering i.e if empty display email else username.